chain-vote/src/committee.rs

```diff
@@ -124,6 +124,11 @@ impl MemberSecretKey {
     pub fn to_bytes(&self) -> [u8; 32] {
         self.0.sk.to_bytes()
     }
+
+    pub fn from_bytes(bytes: &[u8; 32]) -> Option<Self> {
```

Comment:
I think it's more convenient to take a dynamically-sized slice, like in the other places.

And all these places badly need error reporting. `Option` is a wrong way to indicate bad input data.